app/addons/documents/index-results/stores.js

```diff
@@ -472,19 +474,9 @@ Stores.IndexResultsStore = FauxtonAPI.Store.extend({
   },
 
   getTableViewData: function () {
-    var res;
-    var schema;
-    var hasIdOrRev;
-    var hasIdOrRev;
-    var prioritizedFields;
-    var hasBulkDeletableDoc;
-    var database = this.getDatabase();
-    var isView = !!this._collection.view;
-
     // softmigration remove backbone
-    var data;
-    var collectionType = this._collection.collectionType;
-    data = this._filteredCollection.map(function (el) {
+    const collectionType = this._collection.collectionType;
+    let data = this._filteredCollection.map(function (el) {
```

Comment:
this could become `let data = this._filteredCollection.map(el => fixDocId.....)`